### DISCUSSION TITLE
How to change the color of the horizontal split sepparator

### DISCUSSION THREAD

**Discussion by fingust** [olimorris/onedarkpro.nvim#218] [Q&A] [answered]
By default the color of the separator in vertical splits is the same as the background color of the buffers, which makes it really hard to see where one split ends and another one starts.

E.g. this is how it looks in this theme:
![Screenshot_20231104_001138](https://github.com/olimorris/onedarkpro.nvim/assets/18039822/94175aec-9480-40e4-b2f4-ff662e7ce3c8)

And this is how it looks in the "regular" version of this theme (navarasu/onedark.nvim):
![Screenshot_20231104_001031](https://github.com/olimorris/onedarkpro.nvim/assets/18039822/35c3f089-35c5-45a9-accc-03b48d00d35a)

How can I change the color of that separator?

**Comment by mmirus** [comment, marked as answer]
I believe that should be `StatusLineNC`, but that may not apply if you're using a status line plugin like lualine.
